webui/react/src/shared/utils/datetime.ts

```diff
@@ -1,12 +1,15 @@
 import dayjs from 'dayjs';
 import utc from 'dayjs/plugin/utc';
 
+// TODO: check types for this.
+// @ts-ignore
+import humanizeDuration from 'humanize-duration';
+
 import { StartEndTimes } from 'types';
 
 dayjs.extend(utc);
 
 /* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
question: is this line still needed with the removal of the "require"? awesome that we can conform to `import`!